src/runtime/graph/graph_runtime.cc

```diff
@@ -418,6 +419,9 @@ PackedFunc GraphRuntime::GetFunction(const std::string& name,
       int in_idx = 0;
       if (args[0].type_code() == kTVMStr) {
         in_idx = this->GetInputIndex(args[0]);
+      } else if (args[0].IsObjectRef<runtime::String>()) {
+        auto str = args[0].AsObjectRef<runtime::String>();
+        in_idx = this->GetInputIndex(str);
```

Comment:
This is kind of verbose that we have to check kTVMStr and Object runtime::String...Is there any better solution?

Reply:
Perhaps we can introduce `static runtime::String::CanConvertFrom(TVMArgValue)` and `runtime::String str = args[0];`

Reply:
Is this saying we overload "operator T" of `TVMArgValue`? Do we put it in container.h or packed_func.h?

Reply:
We already overloaded operator T, just need to check the type.

Reply:
```CanConvertFrom ``` is a check to check if the TVMArgValue can be converted to string